### ISSUE TITLE
Feature Request:  retain audio/subtitle settings for next episode when they were set by the previous one.

### ISSUE THREAD

**Issue by aok-dev** [MediaBrowser/Emby#3677]
The Issue:

I start watching Show A - Episode 1.  I notice the audio/subtitle settings aren't what I want - so I used the CC button and the Audio link to change them.  Episode finishes, and Episode 2 starts - but now the settings are back to what they were and are not my updated settings.

There's not a place to set them before starting to watch, or a place to do it on the main Show screen or the Season screen.

Request:

Provide a way to set them by show/season - OR retain them once I set them on one episode for remaining episodes.

**Comment by LukePulverenti** [comment]
Duplicate of #3605

**Comment by LukePulverenti** [comment]
FYI: https://github.com/MediaBrowser/Emby/issues/3605
